<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NTK0260" failures="5" tests="8" name="Test" time="18.157" errors="0" timestamp="2022-06-24T22:03:06 ART">
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="2.752"/>
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="8.029"/>
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="0.663">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Edit.SauceDemo.Asignacion7.generarOrden(Asignacion7.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- generarOrden -->
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="0.527">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Edit.SauceDemo.Asignacion7.generarOrden(Asignacion7.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- generarOrden -->
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="0.518">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Edit.SauceDemo.Asignacion7.generarOrden(Asignacion7.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- generarOrden -->
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="2.118"/>
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="0.501">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Edit.SauceDemo.Asignacion7.generarOrden(Asignacion7.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- generarOrden -->
  <testcase classname="Edit.SauceDemo.Asignacion7" name="generarOrden" time="1.346">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/checkout-step-one.html] but found [https://www.saucedemo.com/checkout-step-one.html]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/checkout-step-one.html] but found [https://www.saucedemo.com/checkout-step-one.html]
at Edit.SauceDemo.Asignacion7.generarOrden(Asignacion7.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- generarOrden -->
</testsuite> <!-- Test -->
